tuning/data/setup_dataprocessor.py

```diff
@@ -114,6 +114,15 @@ def process_dataconfig_file(
     )
 
     if data_processor.processor_config.streaming:
+        if getattr(train_args, "padding_free", None):
```

Comment:
you have to use is_padding_free
its passed here https://github.com/foundation-model-stack/fms-hf-tuning/blob/main/tuning/data/setup_dataprocessor.py#L491
there's no padding_free in train_args so your code has a bug

Reply:
actually it won't be a bug ig as if its not present it wont throw an error, but if it is passed it ensures the if condition.